### ISSUE TITLE
Need help with install

### ISSUE THREAD

**Issue by ardu111** [JonnyPtn/zomboi#86]
I am sorry but i am total newbie in linux i got a vps and working zomboid server installed by https://pzwiki.net/wiki/Dedicated_server. Its on ubuntu 22.0.4. 

I menage to get work your script/bot from main and server commands was working from discord but the problem is that its forcing making new server in /root/zomboid/ and i am fighting this almost 2 days and cant get it to use my folder in /home/pzuser/Zomboid for config saves can u please explain me this like for a idiot i am not really famillar with linux still learning and i want really use it because commands on discord was working and i dont want to give vps password for my subadmins.. THanks for reading this and i am sorry for my grammar.

Wher i should put all your main files i use dotnet run from /opt/

**Comment by Fallout001** [comment]
I too need assistance with install, if this project is still in development would it be possible to add more installation instructions like where to place files, what files will be used, what proper configuration looks like, and etc. im having trouble getting started with it

**Comment by GameAreaCZ** [comment]
Wasn't this bot python bot in the past? That was better because you can use it dedicated to the server.

**Comment by thefyLoX** [comment]
> Wasn't this bot python bot in the past? That was better because you can use it dedicated to the server.

Yes, it's still up on the master branch. There are forks based on the python version as well.

I'm still reluctant to swap my python bot for the C# one, at least until it gets more documentation.
